src/main/resources/interactables.yml

```diff
@@ -95,6 +107,7 @@ interactables:
   - 'ENCHANTING_TABLE'
   - 'ENCHANTMENT_TABLE'
   - 'ENDER_CHEST'
+  - 'ENDER_PEARL'
```

Comment:
I don't think this has any effect as only blocks are checked against the interactables list.
Other than that, looks good